nifi-nar-bundles/nifi-extension-utils/nifi-prometheus-utils/src/main/java/org/apache/nifi/prometheus/util/PrometheusMetricsUtil.java

```diff
@@ -364,6 +370,10 @@ public static CollectorRegistry createConnectionStatusAnalyticsMetrics(Connectio
         return connectionAnalyticsMetricsRegistry.getRegistry();
     }
 
+    private static int getUtilization(final double used, final double total) {
```

Comment:
The Gauge value is a `double` so this will get turned back into a double, should we just keep the calculation as a double instead?

Reply:
yep not a problem, this can return double. I'll update it.